### DISCUSSION TITLE
How to reset to default access point

### DISCUSSION THREAD

**Discussion by rsalesas** [RavenSystem/esp-homekit-devices#1946] [Q&A] [answered]
Is there any way to have the device go back to its default access point? I seem to have entered an incorrect password, and it won't connect to my wifi network. I tried restarting it within 2 seconds, but that doesn't seem to do anything.

**Comment by freedreamer82** [comment]
@RavenSystem  any solution , proposal ? Let's image to change your router...and your wifi according to that. How to reset devices to go  in access point mode ? I think we should need a local button to force this mode.

what do u think?

**Reply by xrust83** [reply]
https://github.com/RavenSystem/esp-homekit-devices/wiki/Setup-Mode#entering-setup-mode
https://github.com/RavenSystem/esp-homekit-devices/wiki/Setup-Mode#change-wifi-network

**Comment by freedreamer82** [comment]
I know how to enter in setup… but the case i described is different.The assumption is that u can reach the device… but if your network is not There anymore? How can u restart The process From the AP mode?

**Comment by RavenSystem** [comment, marked as answer]
As @xrust83 says, from the Wiki page https://github.com/RavenSystem/esp-homekit-devices/wiki/Setup-Mode#entering-setup-mode :

> If the device's configured WiFi network is unavailable when entering setup mode it will switch to AP mode and you will need to connect directly to it via the generated SSID

**Comment by freedreamer82** [comment]
Perfect now is clear , thanks!
